src/components/Comments.jsx

```diff
@@ -19,6 +23,19 @@ function Comments({ article_id }) {
             })
     }, [article_id, triggerReloadComments])
 
+
+    function handleDeleteComment(comment_id) {
+        setCommentDeleteStatus({ comment_id: Number(comment_id), message: "Deleting comment..." })
+        deleteArticleComment(comment_id)
+            .then((response) => {
+                setTriggerReloadComments((current) => current + 1)
+                setCommentDeleteStatus({ comment_id: null, message: "" })
+            })
+            .catch(error => {
+                setCommentDeleteStatus({ comment_id: Number(comment_id), message: "Oops... there was a problem deleting the comment" })
+            })
+    }
```

Comment:
Great that you're informing the user if their comment deletion hasn't worked via an error message 👍 
It's not ideal that you are reloading the comments from the API after every comment is added, though on the upside it would mean you see comments others have used without refreshing. However it does negate the benefit of updating state and it efficiently just re-rendering components as necessary